### ISSUE TITLE
navigation drop-down unexpected at narrow layout

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#250]
## Report Issue
At narrower layouts, the navigation menu items lack of distinction between direct links to a new page and items that open up a drop down make the menu confusing.

## Preferred Solution
a visual distinction between menu items that will drop down and those which link to a page for example a triangle before the menu item is often used to show it has submenus. This should then have an appropriate aria-label to make the same distinction for assistive tech
